.github/pull_request_template.md

```diff
@@ -1,35 +1,34 @@
-<!-- Add an assignee and use `component:` or version labels for good hygiene! -->
+<!-- Relate or link the associated epic or task. -->
+<!-- Add `@camunda/tech-writers` as reviewer to pull in a tech writer, or add your embedded tech writer. -->
 
 ## When should this change go live?
 
 <!-- PRs merged go to stage.docs.camunda.io first and must be manually released to docs.camunda.io. -->
 
-- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
-- [ ] This is already available but undocumented and should be released within a week.
-- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
-- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
-- [ ] There is **no urgency** with this change and can be released at any time.
+- [ ] This is a bug fix, security concern, or something that needs **urgent release support**. (apply `bug` or `support` label)
+- [ ] This is already available but undocumented and should be released within a week. (apply `available & undocumented` label)
+- [ ] This is on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (create draft PR and/or apply `hold` label)
+- [ ] This is part of a scheduled **alpha or minor**. (apply alpha or minor label)
+- [ ] There is **no urgency** with this change (add `low prio` label)
 
 ## PR Checklist
 
-<!-- Camunda maintains 18 months of versions. Backporting your change to multiple versions is common. -->
+<!-- Camunda maintains 18 months of minor versions. Backporting your change to multiple versions is common. -->
 
-- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
 - [ ] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).
+- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
 
 <!-- UNCOMMENT THIS SECTION IF APPLICABLE. Adding or removing pages requires extra steps.
 - [ ] I included my new page in the sidebar file(s).
 - [ ] I added a redirect for a renamed or deleted page to the .htaccess file.
 -->
 
-<!-- All changes require either an Engineering review or technical writer review. **Many require both!** -->
-
-- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
-- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.
+- [ ] I added a DRI, team, or delegate as a reviewer for technical accuracy and grammar/style:
+  - [ ] [engineering team review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process)
```

Comment:
This is a nitpick 🥴 

```suggestion
  - [ ] [Engineering team review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process)
```

Reply:
Oddly enough, I went back and forth on this 😅 . I was thinking we want to capture the relevant team reviewer, not the "royal" Engineering org. I don't think people will infer that from capitalization like I _might_, so I'm happy to correct this.